package.json

```diff
@@ -39,21 +39,26 @@
     "discord-oauth2": "^2.11.0",
     "dotenv": "^16.0.3",
     "dotenv-flow": "^3.2.0",
+    "envalid": "^7.3.1",
     "express": "^4.18.2",
     "express-async-router": "^0.1.15",
+    "express-response-errors": "^1.0.5",
     "jsonwebtoken": "^9.0.0",
     "lodash": "^4.17.21",
     "next": "^12.1.2",
+    "next-absolute-url": "^1.2.2",
     "pg": "^8.9.0",
     "pg-hstore": "^2.3.4",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "sequelize": "^6.29.0",
     "tsconfig-paths": "^4.1.2",
     "typescript": "^4.9.4",
+    "umzug": "^3.2.1",
```

Comment:
This is sequelize's tool for running migrations programmatically.  We need to do this in order to run tests on a fresh database (with migrations). [NPM link](https://github.com/sequelize/umzug#umzug)